examples/example_files/off_files/gas_sd.imd

```diff
@@ -0,0 +1,87 @@
+TITLE
+    SD in vacuum
+END
+SYSTEM
+#      NPM      NSM
+         1        0
+END
+STOCHDYN
+#     NTSD     NTFR     NSFR   NBREF  RCUTF    CFRIC    TEMPSD
+         1        1      100       5    0.3     91.0    298
+END
+COVALENTFORM
+#    NTBBH    NTBAH     NTBDN
+             1        1         0
+END
+STEP
+#   NSTLIM         T        DT
+     20000         0     0.002
+END
+BOUNDCOND
+#      NTB    NDFMIN
+         0         3
+END
+FORCE
+#      NTF array
+# bonds    angles   imp.     dihe     charge nonbonded
+  0        1        1        1        1        1
+# NEGR    NRE(1)    NRE(2)    ...      NRE(NEGR)
+     1      18
+END
+CONSTRAINT
+# NTC
+    3
+#      NTCP  NTCP0(1)
+          1    0.0001
+#      NTCS  NTCS0(1)
+          1    0.0001
+END
+PAIRLIST
+# algorithm    NSNB   RCUTP   RCUTL    SIZE    TYPE
+          0       5     1.4     1.4     0.4       0
+END
+NONBONDED
+# NLRELE
+         1
+#  APPAK    RCRF   EPSRF    NSLFEXCL
+         0       1.4        4.0         1
```

Comment:
I think EPSRF should be 1 in vacuum

Reply:
thanks for this input. Should not have a big influence but I changed it